dev/make/compiler_definitions/icx.mkl.32e.mk

```diff
@@ -18,29 +18,40 @@
 #  Intel compiler definitions for makefile
 #--
 
-PLATs.icx = lnx32e mac32e
+PLATs.icx = lnx32e win32e
 
-CMPLRDIRSUFF.icx = _icx
+CMPLRDIRSUFF.icx =
```

Comment:
should it be empty?

Reply:
Its the question how our release folder should look: __release_lnx or __release_lnx_icx